src/common/http.js

```diff
@@ -30,25 +30,31 @@ axios.interceptors.request.use(
 )
 
 axios.interceptors.request.use(async (config) => {
-  if (store.state.request_queue) {
-    //nothing
-  } else {
-    NProgress.start()
+  if (!config.doNotEffectProgress) {
+    if (store.state.request_queue) {
+      //nothing
+    } else {
+      NProgress.start()
+    }
```

Comment:
Much better than `nothing` comment?
```suggestion
    if (!store.state.request_queue) {
      NProgress.start()
    }
```

Reply:
yep